internal/ais/workflow.go

```diff
@@ -154,10 +168,10 @@ func (w *Workflow) SessionHandler(ctx temporalsdk_workflow.Context, params *Work
 	if parseResult.UpdatedAreldaMetadataRelPath != "" {
```

Comment:
Instead of this conditional why not do something like:
```go
var metadataRelPath string
if parseResult.UpdatedAreldaMetadataRelPath != "" {
    metadataRelPath = parseResult.UpdatedAreldaMetadataRelPath 
} else {
    metadataRelPath = parseResult.MetadataRelPath
}

if metadataRelPath != "" {
    var fetchMetadataResult FetchActivityResult
    e = temporalsdk_workflow.ExecuteActivity(
    ...
}
```

That last conditional is probably unnecessary, we should never have a SIP without one or the other metadata file.

Reply:
Good call! I'll change the last condition and return an error if no metadata file was found.